DataFormats/DeepFormats/src/classes_def.xml

```diff
@@ -0,0 +1,30 @@
+<lcgdict>
+
+  <class name="btagbtvdeep::JetFeatures" ClassVersion="10">
+    <version ClassVersion="10" checksum="386672900"/>
```

Comment:
please use v 3 for all new classes

Reply:
sure

Reply:
fixed with https://github.com/cms-sw/cmssw/pull/19893/commits/347c96268c4f6f017384998cee2bdee31372fa64